src/libraries/System.Linq/src/System/Linq/Max.cs

```diff
@@ -18,6 +18,9 @@ public static partial class Enumerable
             public static bool Compare(T left, T right) => left > right;
             public static Vector128<T> Compare(Vector128<T> left, Vector128<T> right) => Vector128.Max(left, right);
             public static Vector256<T> Compare(Vector256<T> left, Vector256<T> right) => Vector256.Max(left, right);
+#if NET8_0_OR_GREATER
```

Comment:
You shouldn't need the `#if`s since `System.Linq` is only using the latest TFMs.

Reply:
Oh yes, you are right.. Removed them. Didn´t knew this, but it makes sense.
Fixed